docs/release_notes/2023/apr2023.md

```diff
@@ -41,7 +41,6 @@ Data quality refers to the accuracy, relevance, completeness, and consistency of
 There are several factors that affect data quality. These include data entry errors, data duplication, missing data, inconsistent data, and outdated data. These issues can arise from human error, system errors, or technical issues such as data integration problems.
 
 In Prophecy, you, as a user, can now easily set **Expectations** on your data such as uniqueness, range, and more. You can then schedule these **Expectations** to run at fixed **Schedules**, and receive alerts whenever any mismatch occurs, empowering you to maintain the quality of your data.
```

Comment:
I suggest removing this section #### Data Quality (_Beta_)

Reply:
I was hesitant on completely removing items from published release notes, but I'll go ahead with your suggestion.